LICENSE.md

```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2015 SmartyStreets
+Copyright (c) 2016 Google Inc.
```

Comment:
If you want all contributions to have their copyright assigned to SmartyStreets, please say so upfront in a `CONTRIBUTING.md` document in your repository.

This is not removing SmartyStreets. It's just listing Google as an *additional* copyright holder for the added changes.